src/docs/product/integrations/issue-tracking/jira/index.mdx

```diff
@@ -158,6 +158,24 @@ If you hit a 4xx or 5xx error during or after setting up the Jira Server integra
 
 Alert notifications in Sentry can be routed to many supported integrations, but by default are aimed at email. You will need to configure a project’s [**Alert Rules**](/product/alerts-notifications/alerts/) to properly route notifications to a specific integration.
 
+### Ignored Fields
+
+Ignored fields are Jira fields that are hidden in the issue creation form. To add ignored fields, navigate to **Organization Settings > Integrations** and click "Configure" next to your Jira instance. Then scroll down to “Ignored Fields”.
+
+Type the id or name of the Jira field, not the label, into the text input box. The following section will state how to find the field id or name.
+
+#### Find the id of the Field
```

Comment:
I'd suggest linking to an existing Atlassian doc about this like [this](https://confluence.atlassian.com/jirakb/how-to-find-id-for-custom-field-s-744522503.html) (that was the first one I found, I didn't dig too deeply so there might be a better one) instead of having this section. Then the "The following section will state how to find the field id or name." sentence can be something like "If you need help finding the field's ID, see the Atlassian docs here".

Reply:
@ceorourke thank you for your feedback! I really appreciate it!

In regards to your first point - I mostly used the [link](https://confluence.atlassian.com/jirakb/how-to-find-id-for-custom-field-s-744522503.html) you mentioned, but I added step 4 (Click the “Select an element” button on the top left of the Inspect module) as I found that I couldn't get the correct ID just from clicking on it (see image below). This may just be an issue on my end though. If you are able to, could you try it and let me know if it worked for you?
<img width="1728" alt="6B83215D-2740-4B3E-8945-EE18D13A0D6E" src="https://user-images.githubusercontent.com/116035587/202266121-0e37a382-8196-469d-ac1a-a130bcac8080.png">

In regards to your second point, yes they are the same thing. I've seen it being called both a ["name"](https://getsentry.atlassian.net/browse/WOR-2280) and ["id"](https://confluence.atlassian.com/jirakb/how-to-find-id-for-custom-field-s-744522503.html). In a discussion with the docs team, I got the feedback to call it "id or name" so that it covers both cases, but if you think one makes more sense I can change it as I can see the confusion.

Reply:
<img width="161" alt="Screen Shot 2022-11-16 at 12 44 02 PM" src="https://user-images.githubusercontent.com/29959063/202290094-8a62097b-61bf-4a1c-8664-09b7301747b9.png">

I tried this out and I think it has to do with very specific clicking 😅 If you get the green div it will have the customfield but if you get the blue one it doesn't. The added steps 4 and 5 were confusing until I realized this.

Reply:
Yeah I think generally linking to a 3rd party if possible is better than having to maintain our own docs in case things change. 

Regarding the id or name thing, what's confusing to me is that "name" and "label" can be easily conflated, so we're telling them to not use the label but use the name ... as a user I wouldn't know what the difference is, so personally I'd go with ID but I wouldn't block this from going through on this point.

Reply:
Thank you for your response! I made the changes you suggested.